### ISSUE TITLE
Unable to open Microsoft edge browser even after adding edge driver

### ISSUE THREAD

**Issue by vshivaku** [florentbr/SeleniumBasic#248]
I have added edge diver to SeleniumBasic path

I am calling finely in my VBA as below

```
Set driver = New Selenium.EdgeDriver
driver.Start "edge"
```

when we call driver.Start it suppose to open the bowser but it is not opening. 
I followed steps clearly in https://florentbr.github.io/SeleniumBasic/. can you please check what could be issue.
Note : The same configuration working fine for chrome browser. with lot of research and work around only raising this issue.

Thanks in advance

**Comment by GCuser99** [comment]
I don't have any trouble starting Edge that way. My current browser and driver version is 100.0.1185.44.

**Comment by vshivaku** [comment]
Is there any version compatibility issue. below are versions config i have
Edge driver : 102.0.1227.0
Microsoft Edge Version 100.0.1185.44
any thing wrong with the versions?

**Comment by GCuser99** [comment]
I don't know if those are compatible, but the general rule is that the major version (the number to the left of the first period) should match for compatibility. So try with an Edge driver version 100.X.XXX.XX.

**Comment by vshivaku** [comment]
Yes that`s correct. tried this 100.X.XXX.XX, its working. Thanks.
Is it possible same with firefox?

**Comment by GCuser99** [comment]
The general rule applies for all driver/browser pairs.

**Comment by GCuser99** [comment]
That general rule applies to Chrome and Edge - I don't know about Firefox.

**Comment by vshivaku** [comment]
Actually i am making to open default browser of system. in that process achieved for chrome and edge. For Firefox I didn`t find the driver. So is there any source or solution for this?

**Comment by GCuser99** [comment]
this may be of some help:
https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html

**Comment by vshivaku** [comment]
Thanks @GCuser99 I will work on this.

**Comment by mogulman52** [comment]
There is no solution for FF.  FF changed how they access DOM.
